src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -1982,51 +1981,51 @@ namespace AirflowNetwork {
                 } else if (SELECT_CASE_var == "MULTIZONEWITHDISTRIBUTIONONLYDURINGFANOPERATION") {
-                    SimulateAirflowNetwork = AirflowNetworkControlSimpleADS;
+                    simulation_control.type = ControlType::MultizoneWithDistributionOnlyDuringFanOperation;
                     SimAirNetworkKey = "MultizoneWithDistributionOnlyDuringFanOperation";
-                } else if (SELECT_CASE_var == "MULTIZONEWITHDISTRIBUTION") {
-                    SimulateAirflowNetwork = AirflowNetworkControlMultiADS;
+                } else { // if (SELECT_CASE_var == "MULTIZONEWITHDISTRIBUTION") {
+                    simulation_control.type = ControlType::MultizoneWithDistribution;
                     SimAirNetworkKey = "MultizoneWithDistribution";
-                } else { // Error
-                    // Code will never be executed, validation will catch invalid input
-                    ShowSevereError(m_state,
-                                    format(RoutineName) + CurrentModuleObject + " object, The entered choice for " + cAlphaFields(2) +
-                                        " is not valid = \"" + AirflowNetworkSimu.Control + "\"");
-                    ShowContinueError(m_state,
-                                      "Valid choices are \"NO MULTIZONE OR DISTRIBUTION\",\"MULTIZONE WITH DISTRIBUTION ONLY DURING FAN OPERATION\"");
-                    ShowContinueError(m_state, "\"MULTIZONE WITH DISTRIBUTION\", or \"MULTIZONE WITHOUT DISTRIBUTION\"");
-                    ShowContinueError(
-                        m_state, "..specified in " + CurrentModuleObject + ' ' + cAlphaFields(1) + " = " + AirflowNetworkSimu.AirflowNetworkSimuName);
-                    ErrorsFound = true;
                 }
             }
         }
 
+        // Determine a convenience boolean or two to simplify the checking
+        // The first one is true if distribution is simulated, replaces some > and >= comparisons
+        // SimulateAirflowNetwork > AirflowNetworkControlMultizone -> type == ControlType::MultizoneWithDistributionOnlyDuringFanOperation
+        //                                                            type == ControlType::MultizoneWithDistribution
+        // SimulateAirflowNetwork >= AirflowNetworkControlSimpleADS -> type == ControlType::MultizoneWithDistributionOnlyDuringFanOperation
+        //                                                             type == ControlType::MultizoneWithDistribution
+        distribution_simulated = simulation_control.type == ControlType::MultizoneWithDistributionOnlyDuringFanOperation ||
+                                 simulation_control.type == ControlType::MultizoneWithDistribution;
+        // This one is true if the multizone simulation is ALWAYS done
+        multizone_always_simulated = simulation_control.type == ControlType::MultizoneWithDistribution ||
+                                     simulation_control.type == ControlType::MultizoneWithoutDistribution;
+
```

Comment:
I've added a couple of convenience booleans to simplify some of the logic. This can be inconvenient if one forgets to add them to the clear_state functions (oops) but since these were getting checked a lot it makes sense to use a single boolean for readability.